public/index.html

```diff
@@ -11,6 +11,23 @@
 </head>
 
 <body>
+    <script type = "text/javascript">
+        const {remote} = require('electron')
+        function isDev() {
+            return remote.process.argv[2] == 'dev';
+        }
+
+        if(isDev()){
+            document.addEventListener("keydown", function (e) {
+                if (e.which === 123) {
+                    remote.getCurrentWindow().toggleDevTools();
+                } else if (e.which === 116) {
+                    location.reload();
+                }
+            });
+            
+        }
+     </script>
```

Comment:
There are a few things wrong with this approach:
* We are coding outside of our source directory, which means that Webpack's rules don't apply here, and this code is treated differently.
* F12 is a problematic key to handle, due to laptop keyboards usually having brightness / volume controls overloaded there. Also, I think that `Fn + F12` might send different key codes depending on the host OS, but don't quote me on that.
* Since this listener is defined in the renderer process, if the native menu bar has focus, we'll miss the event.